src/FSharp.Data.DesignTime/FSharp.Data.DesignTime.fsproj

```diff
@@ -1,13 +1,22 @@
-﻿<Project>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
   <Import Project="..\..\netfx.props" />
   <PropertyGroup>
-    <TargetFrameworks>net45; netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462; netstandard2.0</TargetFrameworks>
     <DefineConstants>NO_GENERATIVE;$(DefineConstants)</DefineConstants>
     <OtherFlags>--warnon:1182</OtherFlags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="..\..\paket-files\fsprojects\FSharp.TypeProviders.SDK\src\ProvidedTypes.fsi">
```

Comment:
because we switched back to paket we can have it manage our file includes as well. see the `paket.references` for details.